doc/user/content/sql/appendix-cluster-sizes.md

```diff
@@ -7,8 +7,40 @@ menu:
     weight: 900
 ---
 
+## Default Cluster Sizes
+
 {{% self-managed/materialize-cluster-sizes %}}
 
+## Custom Cluster Sizes
+
+When installing the Materialize Helm chart, you can override the [default
+cluster sizes and resource allocations](/sql/appendix-cluster-sizes/). These
+cluster sizes are used for both internal clusters, such as the `system_cluster`,
+as well as user clusters.
+
+{{< tip >}}
+
+In general, you should not have to override the defaults. At minimum, we
+recommend that you keep the 25-200cc cluster sizes.
+
+{{</ tip >}}
+
+```yaml
+operator:
+  clusters:
+    sizes:
+      <size>:
+        workers: <int>
+        scale: <int>
```

Comment:
Do we want to let users set scale to anything but 1? Larger scales aren' much tested and might come with caveats (network bandwidth requirements) that aren't well documented.

Reply:
👍  Will update to have `scale: 1     # Generally, should be set to 1.`

We do show, however, in our default settings for `6400cc` a scale value of `2`. Hope that's okay ... since it is what it is set to. https://preview.materialize.com/materialize/32444/self-managed/v25.1/sql/appendix-cluster-sizes/#default-cluster-sizes